domains/authorization/Authorization.API/API/Authorization/Controllers/ConsentController.cs

```diff
@@ -53,9 +53,39 @@ public async Task<ActionResult> GetConsent()
         var queryResult = await mediator.Send(new GetUserOrganizationConsentsQuery(identity.Subject.ToString(), identity.OrganizationCvr!));
 
         var response = new UserOrganizationConsentsResponse(
-            queryResult.Result.Select(item => new UserOrganizationConsentsResponseItem(item.ClientName, item.ConsentDate))
+            queryResult.Result.Select(item => new UserOrganizationConsentsResponseItem(item.ClientId, item.OrganizationId, item.ClientName, item.ConsentDate))
```

Comment:
ClientId is not really usefull, the IdpClientId field in the Client table is probably what you are looking for?

OrganizationId is probably not needed when the delete endpoint is changed to use cvr.